### PR TITLE
Add tests for Athens Cairo SVG and PDF creation + some renames in examples

### LINKED ISSUE #6382: Add tests to Athens-Cairo SVG and PDF creation
**Describe the request**
My PR was merged with #6376 with examples as manual tests. Now we can add some automatic tests to assert that a non-empty file is generated and then delete it should be easy. 

**Expected behavior**
Will call the examples.

**Expected development cost**
Small

### PR DESCRIPTION
**Pull request by tinchodias**
Fixes #6382 and #6383

### CODE REVIEW

**Comment by tinchodias** [comment]
I'm adding tests that export files so it could happen that there is an error in a specific OS like happened in jenkins. BUT the log shows an unrelated error.

**Comment by Ducasse** [comment]
martin one of you setup or teardown is missing a super call.
Error

an OrderedCollection(AthensCairoSurfaceExportTest>>#tearDown) should have been empty
Stacktrace

TestFailure
an OrderedCollection(AthensCairoSurfaceExportTest>>#tearDown) should have been empty
ProperlyImplementedSUnitClassesTest(TestAsserter)>>assert:description:resumable:
ProperlyImplementedSUnitClassesTest(TestAsserter)>>assert:description:
ProperlyImplementedSUnitClassesTest(TestAsserter)>>assertEmpty:
ProperlyImplementedSUnitClassesTest>>testAndMakeSureSuperTearDownIsCalledAsLastMessageInTearDownMethodsOfTestCases
ProperlyImplementedSUnitClassesTest(TestCase)>>performTest

**Comment by tinchodias** [comment]
Thanks, I added a super setUp and a tearDown. Didn't know it is mandatory.

**Comment by Ducasse** [comment]
I do not remember why :)

**Comment by Ducasse** [comment]
I hope that this is ok in Santiago.

File: src/Athens-Cairo-Tests/AthensCairoSVGSurfaceExportTest.class.st [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Class {
+	#name : #AthensCairoSVGSurfaceExportTest,
+	#superclass : #AthensCairoSurfaceExportTest,
+	#category : #'Athens-Cairo-Tests-Core'
+}
+
+{ #category : #tests }
+AthensCairoSVGSurfaceExportTest >> athensSurfaceClass [
+	^ AthensCairoPDFSurface
```
Comment (tom95) [inline src/Athens-Cairo-Tests/AthensCairoSVGSurfaceExportTest.class.st:9]: is this potentially a typo? (as opposed to AthensCairoSVGSurface, if that exists?)

**Comment by Ducasse** [comment]
Martin?
Tom I did not get your typo.

**Comment by tom95** [comment]
Ah I was just skimming the PR and noticed that the SVG test instantiated a PDF surface, rather than a SVG surface. Not sure if that was intentional

**Comment by Ducasse** [comment]
Thanks!
Martin will let us know.

**Comment by tinchodias** [comment]
Sorry, I missed to read your comments.
@tom95 You were right! I will check if it is still there is latest P9 and fix the typo.
@Ducasse for the moment I work from home and don't leave too much. Hope you are fine, too.
Thanks
